examples/scene/scene.rs

```diff
@@ -121,3 +122,9 @@ fn infotext_system(mut commands: Commands, asset_server: Res<AssetServer>) {
         ..Default::default()
     });
 }
+
+fn event_listener_system(mut events: EventReader<SceneSpawnedEvent>) {
+    for my_event in events.iter() {
+        info!("{:?}", my_event.instance_id);
+    }
+}
```

Comment:
I was using this to test this the event is actually getting fired; open to removing it / cleaning it up if that is desirable.

Reply:
You could try adding a test directly in the scene spawner module